tests/unit/Rules/ContainsTest.php

```diff
@@ -16,11 +16,18 @@
 use PHPUnit\Framework\TestCase;
 
 /**
- * @group  rule
+ * @group rule
+ *
  * @covers \Respect\Validation\Rules\Contains
  * @covers \Respect\Validation\Exceptions\ContainsException
```

Comment:
🔪 We're testing the exceptions in the integration tests

Reply:
I got it!